service/controller/key/key.go

```diff
@@ -6,7 +6,7 @@ import (
 
 	"github.com/giantswarm/k8smetadata/pkg/label"
 	"github.com/giantswarm/microerror"
-	apiv1alpha3 "sigs.k8s.io/cluster-api/api/v1alpha3"
+	apiv1alpha4 "sigs.k8s.io/cluster-api/api/v1alpha4"
```

Comment:
back in the day in `azure-operator` we started using aliases that did not contain the version. That way whenever we upgraded the dependency, only the import line needed to be changed, rather than several different places throughout the code. What do you think about following the same approach? I mean using `capi "sigs.k8s.io/cluster-api/api/v1alpha4"`

Reply:
Good idea. I did this for `capi`, `capo`, and `capz` now.

Reply:
@fiunchinho I updated CAPZ to v1alpha4 here.

Reply:
I think that's fine, thanks.